sshcode.go

```diff
@@ -21,7 +22,7 @@ import (
 	"golang.org/x/xerrors"
 )
 
-const codeServerPath = "~/.cache/sshcode/sshcode-server"
+const codeServerDir = "~/.sshcode-server"
```

Comment:
I've made this directory change so it is similar to how `.vscode-server` stores itself. Also a lot of other tools do the same - nvm, npm, docker, cocoapods and etc.

Hopefully this will make sense.